sugarcrm/client.py

```diff
@@ -0,0 +1,70 @@
+import requests
+import json
+import hashlib
+import sugarcrm
+from sugarcrm.error import CrmAuthenticationError, CrmResponseError
+
+
+class APIClient:
+    def __init__(self, client=requests, logger=sugarcrm.logger):
+        self.client = client
+        self.session_id = None
+        self.rest_url = sugarcrm.rest_url
+        self.logger = logger
+
+    def _api_request(self, method, args):
+        data = json.dumps(args)
+        args = {
+            "method": method,
+            "input_type": "json",
+            "response_type": "json",
+            "rest_data": data,
+        }
+        self.logger.debug("SugarCRM request to {}, args={}".format(self.rest_url, args))
+        response = self.client.get(self.rest_url, args)
+        self.logger.debug("SugarCRM response: {}".format(response.content))
+        return response
+
+    def login(self):
+        encode = hashlib.md5(sugarcrm.password.encode("utf-8"))
+        encodedPassword = encode.hexdigest()
```

Comment:
this var should be in snake case for consistency, right?

Reply:
Yeah, sorry , it was coopied when I was researching about suiteCRM tests.